### ISSUE TITLE
consider this as a host for package:directories?

### ISSUE THREAD

**Issue by devoncarew** [google/dart-neats#188]
I'm considering building a `directories` package - a package to make it easy to locate application directories (config, cache, settings, ...) in a way that respects the conventions of the various platforms (mac, windows, linux). What do you think about this repo as a potential host for the package?

cc @jonasfj

**Comment by jonasfj** [comment]
Sure, but what do you want it to offer besides what's already in:
https://pub.dev/packages/xdg_directories

Could it not be added to this package?

**Comment by jonasfj** [comment]
Or maybe it's Linux only...

**Comment by devoncarew** [comment]
Yeah, that package is linux only. I'm thinking of one that gives you the best state dir, cache dir, ... for an app, given the app's name and the OS conventions (and on Linux, XDG would be used). See https://github.com/dirs-dev/directories-jvm and https://github.com/dirs-dev/directories-rs.

**Comment by jonasfj** [comment]
This could also be a tools.dart.dev package, well need it in various CLI utils.

We already have some of this logic in package:cli_util

**Comment by jonasfj** [comment]
Closing this as I see no further action.

If you want to host an experimental `directories` package here, they by all means file a PR :rofl: 

I think I mostly just suggested that the logic would fit fine in something like `cli_util`.
